content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,172 @@
+
+- **Added** `AuthorizationPolicy` now supports nested JWT claims.
+  ([Issue #21340](https://github.com/istio/istio/issues/21340))
+
+- **Added** support for client side Envoy secure naming config when trust domain alias is used.
+Fix the multi-cluster service discovery client SAN generation: takes all endpoints' service accounts
+into account, rather than the first found service registry.
+
+- **Added** Experimental feature support allowing Istiod to integrate with external certificate authorities using Kubernetes CSR API (>=1.18 only).
+  ([Issue #27606](https://github.com/istio/istio/issues/27606))
+
+- **Added** Enable user to set the custom VM identity provider for credential authentication
+  ([Issue #27947](https://github.com/istio/istio/issues/27947))
+
+- **Added** action 'AUDIT' to Authorization Policy that can be used to determine which requests should be audited.
+  ([Issue #25591](https://github.com/istio/istio/issues/25591))
+
+- **Added** support for migration and concurrent use of regular K8S tokens as well as new K8S tokens with audience. This feature is enabled by
+default, can be disabled by `REQUIRE_3P_TOKEN` environment variable in Istiod, which will require new tokens with audience. The
+`TOKEN_AUDIENCES` environment variable allows customizing the checked audience, default remains `istio-ca`.
+
+- **Added** `AuthorizationPolicy` now supports a `Source` of type `remoteIpBlocks`/`notRemoteIpBlocks` that map to a new `Condition` attribute called `remote.ip` that can also be used in the "when" clause.  If using an http/https load balancer in front of the ingress gateway, the `remote.ip` attribute is set to the original client IP address determined by the `X-Forwarded-For` http header from the trusted proxy configured through the `numTrustedProxies` field of the `gatewayTopology` under the `meshConfig` when you install Istio or set it via an annotation on the ingress gateway.  See the documentation here: [Configuring Gateway Network Topology](/docs/ops/configuration/traffic-management/network-topologies/). If using a TCP load balancer with the Proxy Protocol in front of the ingress gateway, the `remote.ip` is set to the original client IP address as given by the Proxy Protocol.
+ ([reference](/docs/reference/config/security/authorization-policy/))([usage](/docs/ops/configuration/traffic-management/network-topologies/))([usage](/docs/tasks/security/authorization/authz-ingress/)) ([Issue #22341](https://github.com/istio/istio/issues/22341))
+
+- **Added** Trust Domain Validation by default rejecting requests in sidecars if the request is not from same trust domain
+or if it's not in the `TrustDomainAliases` specified in the `MeshConfig`.
+  ([Issue #26224](https://github.com/istio/istio/issues/26224))
+
+## Telemetry
+
+- **Updated** the "Control Plane Dashboard" and the "Performance Dashboard" to use the `container_memory_working_set_bytes` metric
```

Comment:
Can you add a known issues section with the following content:

1. Memory leak in TCP Wasm extensions affecting TCP telemetry (see https://github.com/istio/istio/issues/24720). The leak occurs when upstream connections are interrupted mid-stream.

2. Wasm extension configuration updates are disruptive (see https://github.com/envoyproxy/envoy/issues/13690). The configuration is immediately applied for existing requests and connections, and is not reverted if the outer xDS is rejected.

cc @mandarjog @douglas-reid

Reply:
@kyessenov I added the known issues section, let me know if that was the format you were looking for

Reply:
I don't think this is right. These two known issues do not belong to this item (`**Updated** the "Control Plane Dashboard"...`). I think we need a top level `Known Issue` section? I'd need to add one for aggregate cluster as well: https://github.com/istio/istio/issues/28620. Do we have any precedence from past releases about where to add it?

Reply:
1.1 and earlier releases have them at the announcement level - but I feel they are better suited for top level of change-notes